_includes/language_search.html

```diff
@@ -1,29 +1,48 @@
 <section class="language-search">
   <div class="container">
-    {% if site.languages.size > 1 %}
-    <div class="language-container">
-      <button class="usa-button usa-button--outline btn-lang-toggle">
-        {% include svg/globe.svg %}
-        <span class="label"> {% t global.language %} </span>
-      </button>
-      <ul class="dropdown">
-        {% for locale in site.languages %} {% capture name %}global.locales.{{
-        locale }}{% endcapture %}
-        <li>
-          {% if locale == 'en' %}
-          <a href="{{ page.url | prepend: site.baseurl_root }}">{% t name %}</a>
-          {% elsif site.lang != 'en' and locale == site.lang %}
-          <a href="{{ page.url | prepend: site.baseurl  }}">{% t name %}</a>
-          {% else %}
-          <a
-            href="{{ page.url | prepend: locale | prepend: '/' | prepend: site.baseurl_root }}"
-            >{% t name %}</a
-          >
-          {% endif %}
-        </li>
-        {% endfor %}
-      </ul>
+    <div class="grid-col-6 display-flex flex-align-center">
+      {% if include.logo %}
+      <div class="usa-logo" id="extended-logo">
+        <a href="{{ '/' | prepend: site.baseurl }}" title="Home" aria-label="Home">
+          <img src="{{ site.baseurl }}/assets/img/logo.svg" class="usa-logo__img" alt="login.gov home" />
+        </a>
+        <span>|</span>
+        <a href="{{ '/' | prepend: site.baseurl }}" class="usa-logo__help">{% t nav.help_center %}</a>
```

Comment:
Should this be linking to the help center? Currently it links to the homepage, which could be redundant (related to [H2: Combining adjacent image and text links for the same resource](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H2)).